sympy/solvers/ode/tests/test_ode.py

```diff
@@ -39,38 +39,17 @@ def _ode_solver_test(ode_examples):
         eq = ode_examples['examples'][example]['eq']
         sol = ode_examples['examples'][example]['sol']
         if our_hint not in classify_ode(eq):
-            message = """Hint did not match the example {example}
-
-                The ODE is:
-                {eq}
-
-                The expected hint was
-                {our_hint}
-                """.format(example=example, eq=eq, our_hint=our_hint)
+            message = """Hint did not match the example {example}.\n\nThe ODE is:\n{eq}.\n\nThe expected hint was\n{our_hint}""".format(example=example, eq=eq, our_hint=our_hint)
```

Comment:
This line is too long. You should use a triple-quoted string but without the indentation e.g.:
```
            message = """\
Hint did not match the example {example}

The ODE is:
{eq}
...
```